pkg/kv/kvserver/replica_raft.go

```diff
@@ -500,8 +404,8 @@ func (r *Replica) propose(
 	//
 	// TODO(tschottdorf): can we mark them so lightstep can group them?
 	const largeProposalEventThresholdBytes = 2 << 19 // 512kb
-	if cmdLen > largeProposalEventThresholdBytes {
-		log.Eventf(p.ctx, "proposal is large: %s", humanizeutil.IBytes(int64(cmdLen)))
+	if len(p.encodedCommand) > largeProposalEventThresholdBytes {
```

Comment:
This has switched from `p.command.Size()` to `len(p.encodedCommand)`. Is this ok? Is this the same size?

Reply:
It's not the same size, but it's comparable. The threshold is pretty arbitrary too.